### DISCUSSION TITLE
OpenDTU bei Heise.de

### DISCUSSION THREAD

**Discussion by sivar2311** [tbnobody/OpenDTU#259] [General]
OpenDTU hat es  beim Thema Einspeiseüberwachung in den [Heise Ratgeber PV](https://www.heise.de/ratgeber/Balkonkraftwerke-Die-haeufigsten-Fragen-zu-Installation-etc-und-die-Antworten-7311273.html) geschafft!

**Comment by madmartin** [comment]
Ja, hab ich auch wohlwollend zur Kenntnis genommen!

**Comment by D3R-ST3FAN** [comment]
Und jetzt auch in der aktuellen ct: Doppelseite OpenDTU.
Gratulation! Heise steht scheinbar drauf ;-)

**Comment by madmartin** [comment]
Freut mich sehr!

**Comment by madmartin** [comment]
Und damit es noch der letzte sieht: jetzt auch auf dem Newsticker!

https://www.heise.de/newsticker/

https://www.heise.de/ratgeber/Balkonkraftwerke-Hoymiles-Wechselrichter-ueberwachen-per-Web-MQTT-7327187.html?wt_mc=intern.red.plus.newsticker.7-tage-news.teaser.teaser
